packages/webpack-extraction-plugin/src/webpackLoader.ts

```diff
@@ -94,14 +90,19 @@ function webpackLoader(
         );
       }, '');
 
-      const request = `import ${JSON.stringify(
-        this.utils.contextify(
-          this.context || this.rootContext,
-          `griffel.css!=!${virtualLoaderPath}!${resourcePath}?style=${toURIComponent(css)}`,
-        ),
-      )};`;
+      const outputFileName = this.resourcePath.replace(/\.[^.]+$/, '.griffel.css');
+
+      const request = `${outputFileName}!=!${virtualLoaderPath}?style=${toURIComponent(css)}!${this.resourcePath}`;
+      const stringifiedRequest = JSON.stringify(this.utils.contextify(this.context || this.rootContext, request));
 
-      this.callback(null, `${result.code}\n\n${request};`, result.sourceMap);
+      this.callback(
+        null,
+        // Heads up!
+        // This is probably a bug, but "import" does not work properly there: files from node_modules are processed, but
+        // they will not be present in chunks 😳
```

Comment:
This costs me 5 hours of debugging in product, will create an issue in Webpack as it's clearly bug in Webpack or in css-loader/mini-css-extract.